app/main/views/sign_in.py

```diff
@@ -97,11 +97,10 @@ def _get_access_token(code, state):  # pragma: no cover
     try:
         access_token = response_json["access_token"]
     except KeyError as e:
-        # Capture the response json here so it hopefully shows up in error reports
         current_app.logger.exception(
-            f"Error when getting access token {response.json()} #notify-admin-1505"
+            f"Error when getting access token {response_json()} #notify-admin-1505"
         )
-        raise KeyError(f"'access_token' {response.json()}") from e
+        raise KeyError(f"'access_token' {response_json()}") from e
```

Comment:
Same with this log statement regarding the intent behind including the original response object:

```suggestion
        raise KeyError(f"'access_token' {response.json()}") from e
```